repos/system_upgrade/el9toel10/actors/krb5conf/convertkrb5conf/libraries/convertpamuserdb.py

```diff
@@ -0,0 +1,21 @@
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api
+from leapp.models import OutdatedKrb5confLocation
+
+
+def _convert_krb5conf(conf_file):
+    with open(conf_file) as f:
+        text = f.read().replace('/etc/ssl/certs/ca-certificates.crt',
+                                '/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem')
+    with open(conf_file, 'w') as f:
+        f.write(text)
```

Comment:
if we are sure that all files wil be always present and `rw` operation will allowed, it's ok. otherwise we should cover it by try-except:
```
try:
     ...
except IOError as e:
    # log error
    return  # give it a chance to fix other files still
```

Reply:
Since the file list is supposed to be collected just before by the `scankrb5conf` actor, I think we can expect the files to still be there at this point.

I just realize that I forgot to rename this file though... :sweat_smile:

Reply:
Is Leapp running as root? If that's the case, I don't expect it to face any write permissions problems. However, if one of the krb5 config file in `/etc/krb5.conf.d/` is not publicly readable (which makes no sense, but is technically possible), it will probably cause the `scankrb5conf` actor to fail. Do you think I should just log a warning and skip the erroneous file in this case?

Reply:
you are right. yes, it is running as root always and as we could read it before, we should be able to read it later as well - DAC is ignored by root, selinux will be always in disabled or permissive mode, and I do not expect someone would set an ACL on these files (like making them immutable). So only thing here would be if a file is removed or renamed during the RPM transaction.